vcluster/manage/backup-restore/volume-snapshots/README.mdx

```diff
@@ -94,12 +94,12 @@ When taking volume snapshots, there are limitations:
 - Volume snapshots cannot be created for PVCs that are not provisioned by CSI drivers with [Volume Snapshots](https://kubernetes.io/docs/concepts/storage/volume-snapshots/) support.
 - Volume snapshots cannot be created for PVCs that are provisioned by CSI drivers that don't support [Volume Snapshots](https://kubernetes.io/docs/concepts/storage/volume-snapshots/).
 - When using host nodes:
-  - Currently volume snapshots are created only for PersistentVolumeClaims that are synced to the vCluster host namespace. This means that, when the [Namespace sync](../../configure/vcluster-yaml/sync/to-host/advanced/namespaces) is enabled, volume snapshots are not created for the PersistentVolumeClaims that are synced to namespaces different than the vCluster host namespace.
+  - Currently volume snapshots are created only for PersistentVolumeClaims that are synced to the vCluster host namespace. This means that, when the [Namespace sync](../../../configure/vcluster-yaml/sync/to-host/advanced/namespaces) is enabled, volume snapshots are not created for the PersistentVolumeClaims that are synced to namespaces different than the vCluster host namespace.
 - When using private nodes:
   - vCluster has to be deployed with `deploy.volumeSnapshotController.enabled` config set to `true`.
   - CSI driver that supports volume snapshots has to be installed in the virtual cluster.
   - The default VolumeSnapshotClass has to be configured for each CSI driver in the virtual cluster. See [Kubernetes Volume Snapshot Classes docs](https://kubernetes.io/docs/concepts/storage/volume-snapshot-classes/) for more details.
-- Volume snapshots are currently not supported for [vCluster standalone](../../deploy/control-plane/binary/).
+- Volume snapshots are currently not supported for [vCluster standalone](../../../deploy/control-plane/binary/).
 
```

Comment:
Can you add a link somewhere on this page, that says "For an example guide please see this walkthrough...." Something like that, and then link to your new page.